app/models/forms/dob.rb

```diff
@@ -1,10 +1,17 @@
 module Forms
   class Dob < Base
-    attribute :date_of_birth, Date
+    include ActiveModel::Validations::Callbacks
+
+    attr_reader :date_of_birth
+    attribute :day_date_of_birth, Integer
```

Comment:
wouldn't be easier to just call these attributes: day, month, year?

Reply:
@leoapost good point. let me try that.